src/DotNetWorker/Hosting/ServiceCollectionExtensions.cs

```diff
@@ -81,6 +84,15 @@ public static IFunctionsWorkerApplicationBuilder AddFunctionsWorkerDefaults(this
             // Converters
             services.RegisterDefaultConverters();
 
+            // Default Json serialization should ignore casing on property names
+            services.Configure<WorkerOptions>(options =>
+            {
+                options.Serializer = new JsonObjectSerializer(new JsonSerializerOptions
```

Comment:
Instead of creating the serializer here (which we defer in the options to avoid creating an instance if the user intends to replace with another serializer), can we just setup a configure call for JsonSerializerOptions and make sure we use that in the WorkerOptions?

e.g.: `services.Configure<WorkerOptions>(...)`
then update WorkerOptions to pull the options if an instance is created.

Reply:
I think you mean... `services.Configure<JsonSerializerOptions>(...)`?

Reply:
Yes, that's what I meant. I stated the right thing in the feedback, but typed the wrong options in the example.

Reply:
Will update the example for clarity.